### PR TITLE
MSAL changelog 4.43.0

### PR DESCRIPTION
**Pull request by pmaytak**
**Changes proposed in this request**
Notes for 4.43.0 plus minor changes to previous release notes.

### CODE REVIEW

**Review by bgavrilMS — APPROVED** [review]

File: changelog.txt [hunk 1]
```diff
@@ -1,54 +1,69 @@
...
+**MSAL.NET Cache Extensions now protects plaintext cache files with owner only read/write permissions**.See [3186](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3186), [169](https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/issues/169).  
+
+### Bug Fixes
+**Client capabilities flags are correctly passed to Android Broker**. See [3203](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3203).  
+**Fixed `WithAccount(result.Account)` to work when using WAM**. See [3121](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3121).   
+**Improved token cache filtering logic**. See [3178](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3178), and [3233](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3233).  
+**Fixed an error in creating UWP package for Microsoft Store upload**.  See [3184](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3184), [3239](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3239).
```
Comment (gladjohn) [inline changelog.txt:12]: This was fixed in 4.42.1

**Review by gladjohn — APPROVED** [review]
looks good, just one comment on the UWP packaging bug

File: changelog.txt (hunk 1 above)
Comment (pmaytak) [reply changelog.txt:12]: I thought it didn't work, so we had to do https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3239?

Comment (gladjohn) [reply changelog.txt:12]: that's correct do we need to mention #3184 on both releases?

Comment (pmaytak) [reply changelog.txt:12]: I think yea, since it's the original issue and it's linked to some other duplicate issues.

**Pull request merged**